src/rendermime/index.ts

```diff
@@ -496,3 +549,58 @@ namespace RenderMime {
     contents: Contents.IManager;
   }
 }
+
+
+/**
+ * The namespace for module private data.
+ */
+ namespace Private {
+  /**
+   * Get the data from a notebook output.
+   */
+  export
+  function getData(output: nbformat.IOutput): JSONObject {
+    let bundle: nbformat.IMimeBundle;
+    switch (output.output_type) {
+    case 'execute_result':
+      bundle = (output as nbformat.IExecuteResult).data;
+      break;
+    case 'display_data':
+      bundle = (output as nbformat.IDisplayData).data;
+      break;
+    case 'stream':
+      let text = (output as nbformat.IStream).text;
+      bundle = {
+        'application/vnd.jupyter.console-text': text
+      };
+      break;
+    case 'error':
+      let out: nbformat.IError = output as nbformat.IError;
+      let traceback = out.traceback.join('\n');
+      bundle = {
+        'application/vnd.jupyter.console-text': traceback ||
+          `${out.ename}: ${out.evalue}`
+      };
+      break;
+    default:
+      break;
+    }
+    return convertBundle(bundle || {});
+  }
+
+  /**
+   * Convert a mime bundle to mime data.
+   */
+  function convertBundle(bundle: nbformat.IMimeBundle): JSONObject {
```

Comment:
Looking at my dependence on this function again, it might simply be enough to expose `getData()`, as this now calls `convertBundle()` before returning. My dependencies all called them in sequence.